app/models/prospective_student.rb

```diff
@@ -86,7 +86,9 @@ def registered_user(user)
       user:                   user,
       unit:                   unit,
       permanently_registered: can_permanently_register?,
-      student_number:         id_number # TODO: must be generated
+      student_number:         id_number, # TODO: must be generated
+      year:                   1, # TODO: can have a different value depending on the characteristics of the program
+      semester:               1
```

Comment:
`year` ve `semester` için default değerleri burada girmek yerine migration dosyasında girsek nasıl olur?
Şu an için sistemde öğrencinin olup olmaması kritik değil farkındayım ama eğer öğrenci varsa `rails db:migrate` hata veriyor. Burayı sonradan prospective student'dan gelecek semester ve year bilgisiyle güncellesek daha iyi olacak.

Reply:
@ecmelkytz Aslında tam olarak default değerler değil, o sebeple migration'da vermek istemedim. Teknik borcu (#1234 ) görmediysen eğer diye açıklıyorum tabi. Migrate hatasını da şimdi farkettim, böylesi daha uygun olur diyorsan değişikliği ekleyeyim?